docs/usage/quickstart.rst

```diff
@@ -247,3 +258,5 @@ services and their configuration.
 .. _Grafana: https://grafana.com
 .. _Cerebro: https://github.com/lmenezes/cerebro
 .. _Kibana: https://www.elastic.co/products/kibana/
+.. _storage: (Link needed)
```

Comment:
This is a TODO?

Reply:
I need to find the way to insert a cross-reference to another file. Currently on it